include/deal.II/cgal/utilities.h

```diff
@@ -60,6 +61,17 @@ namespace CGALWrappers
   using ConcurrencyTag = CGAL::Sequential_tag;
 #  endif
 
+  enum class BooleanOperation
+  {
+    corefine     = 1 << 0, ///< Corefine the two surfaces
+    difference12 = 1 << 1, ///< Compute the boolean difference of the first
```

Comment:
Do we need both? I would have used just one, and always compute the difference between the first and the second, but let's wait to see what others think.